### ISSUE TITLE
logout and getCurrentUserFromServer give unexpected results

### ISSUE THREAD

**Issue by eslamira** [phillwiggins/flutter_parse_sdk#62]
Pubspec : 
```
parse_server_sdk:
    git:
      url: https://github.com/phillwiggins/flutter_parse_sdk.git
      ref: develop
```

> var user = await ParseUser("eslamira", "eslamirapassword", null).login();

// Success login

and : 

> await ParseUser.currentUser();

returns

> {"className":"_User","objectId":"UdMqp45eTs","createdAt":"2019-01-27T16:05:46.340Z","updatedAt":"2019-01-29T15:15:43.276Z","username":"\"\"eslamira\"\"","password":"\"\"eslamirapassword\"\"","services":null,"ACL":{"*":{"read":true},"UdMqp45eTs":{"read":true,"write":true}},"sessionToken":"\"\"r:9c44da87b21d666881886b6a4794c039\"\""}

then i try

> var user = await (await ParseUser.currentUser).logout();

and : 

> await ParseUser.currentUser();

returns the same object

> {"className":"_User","objectId":"UdMqp45eTs","createdAt":"2019-01-27T16:05:46.340Z","updatedAt":"2019-01-29T15:15:43.276Z","username":"\"\"eslamira\"\"","password":"\"\"eslamirapassword\"\"","services":null,"ACL":{"*":{"read":true},"UdMqp45eTs":{"read":true,"write":true}},"sessionToken":"\"\"r:9c44da87b21d666881886b6a4794c039\"\""}

so ? he didn't logout ?!

**Comment by pcegarra** [comment]
Can you try it in the master branch please? Is the same or its working?

**Comment by eslamira** [comment]
i think the problem that you don't clear cache of ParseUser().currentUser ,,
tired on 1.0.7 and 1.0.10 no logout and master/develop too ,,

and getCurrentUserfromServer ,, is working on 1.0.7 ,1.0.10 and master and no luck with develop

for reference working on master : flutter_parse_sdk-62dbcc0ce305804d2adc5652e847dce7ceeb76fd

**Comment by phillwiggins** [comment]
Will look at now.

**Comment by phillwiggins** [comment]
getCurrentUserFromServer is fixed on develop. Still looking into logout issues. Could be related.

**Comment by phillwiggins** [comment]
Fixed in v1.0.12
